charts/payment-integration/values.yaml

```diff
@@ -0,0 +1,28 @@
+image:
+  repository: ''  # either 'sphereio/commercetools-payone-integration' or 'commercetoolsps/commercetools-paypal-plus-integration/'
```

Comment:
please provide a default value here

Reply:
but there is no default value, they are absolutely different and incompatible